webapp/src/component/layout/Notifications/NotificationsTopBarButton.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  default as React,
+  FunctionComponent,
+  useEffect,
+  useState,
+} from 'react';
+import { Badge, IconButton, styled } from '@mui/material';
+import { useApiQuery } from 'tg.service/http/useQueryApi';
+import { Bell01 } from '@untitled-ui/icons-react';
+import { NotificationsPopup } from 'tg.component/layout/Notifications/NotificationsPopup';
+import { NotificationsChanged } from 'tg.websocket-client/WebsocketClient';
+import { PopoverProps } from '@mui/material/Popover';
+
+const StyledIconButton = styled(IconButton)`
+  width: 40px;
+  height: 40px;
+
+  img {
+    user-drag: none;
+  }
+`;
+
+export const NotificationsTopBarButton: FunctionComponent<{
+  className?: string;
+}> = () => {
+  const [anchorEl, setAnchorEl] = useState<PopoverProps['anchorEl']>(null);
+  const [unseenCount, setUnseenCount] = useState<number | undefined>(undefined);
+
+  const unseenNotificationsLoadable = useApiQuery({
+    url: '/v2/notifications',
+    method: 'get',
+    query: { size: 1, filterSeen: false },
+    options: {
+      enabled: unseenCount === undefined,
+      refetchOnMount: false,
+      keepPreviousData: true,
+    },
+  });
+
+  const handleOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  useEffect(() => {
+    if (unseenCount !== undefined) return;
+    setUnseenCount(
```

Comment:
This could be moved to `onSuccess` callback of the `useApiQuery`:
```ts
  const unseenNotificationsLoadable = useApiQuery({
    url: '/v2/notifications',
    method: 'get',
    query: { size: 1, filterSeen: false },
    options: {
      enabled: unseenCount === undefined,
      refetchOnMount: false,
      keepPreviousData: true,
      onSuccess(data) {
        setUnseenCount(data?.page?.totalElements || prevState)
      }
    },
  });
```